Resume.php

```diff
@@ -0,0 +1,88 @@
+<html>
+  <head>
+    <title> Resume/Projects </title>
+    <link rel="stylesheet" href="style.css?Time=<?php echo microtime()?>" />
+  </head>
+  <body>
+      <h1>
+          Projects/Resume</h1>
+          <div class = "active">
```

Comment:
Right now you're not consistent with whether you put a space before and after the "=" or not (e.g. ` a = b` vs `a=b`). I'd suggest coming up with a standard and sticking with it.

I personally have not seen the space used inside html elements like this. I'm not sure whether or not this would work, but I know that the standard is to say `class="active"`. Note: this isn't true with real coding languages. Like, if you've used java before, putting those extra spaces in there is normal, but html isn't a real coding language, and you normally don't include spaces between an html attribute name and value.